Jenkinsfile.release

```diff
@@ -566,11 +566,20 @@ node('zowe-jenkins-agent-dind-wdc') {
       return isFormalRelease && (releaseArtifacts['docker-amd64'] || releaseArtifacts['docker-s390x'])
     },
     stage            : {
-      withCredentials([usernamePassword(
-        credentialsId: 'ZoweDockerhub',
-        usernameVariable: 'USERNAME',
-        passwordVariable: 'PASSWORD'
-      )]) {
+      withCredentials([
+        usernamePassword(
+          credentialsId: 'ZoweDockerhub',
+          usernameVariable: 'USERNAME',
+          passwordVariable: 'PASSWORD'
+        ),
+        sshUserPrivateKey(
+          credentialsId: 'DockerUserSigningKey',
+          keyFileVariable: 'USERSIGNKEY',
```

Comment:
i want to write the key to a file in a specific place docker uses, so i will try using sshuserprivatekey as the store first, but may change do a different credential store if it isnt the right type to store 3 values in one object: key, keyname, and passphrase.